ompi/mca/pml/ucx/pml_ucx.c

```diff
@@ -737,12 +737,23 @@ static inline ucs_status_ptr_t mca_pml_ucx_common_send(ucp_ep_h ep, const void *
                                                        mca_pml_base_send_mode_t mode,
                                                        ucp_send_callback_t cb)
 {
+#if HAVE_DECL_UCP_TAG_SEND_NBX
+    ucp_request_param_t param = {
```

Comment:
static

Reply:
static can't be used because callback is passed as argument

Reply:
so define it in caller function

Reply:
refactored